packages/bpk-docs/src/components/neo/DocsPageWrapper/DocsPageWrapper.js

```diff
@@ -46,6 +47,9 @@ const contentShape = PropTypes.oneOfType([PropTypes.string, childrenPropType]);
 class DocsPageWrapper extends React.Component {
   static propTypes = {
     title: PropTypes.string.isRequired,
+    // https://github.com/ReactTraining/react-router/issues/5129#issuecomment-302131982
+    // eslint-disable-next-line react/forbid-prop-types
+    router: PropTypes.object.isRequired,
```

Comment:
can't we use a shape here? i.e. `PropTypes.shape({ isActive: PropType.func, replace: PropType.func, push: PropTypes.func })`

Reply:
Yep, will do.